src/componets/Calculator.js

```diff
@@ -1,43 +1,66 @@
 import React from 'react';
+import Btn from './button';
 import './calculator.css';
+import calculate from '../logic/calculate';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
[OPTIONAL] I think you can delete this disable linter rule now that you're using a component with state.